setup.py

```diff
@@ -21,3 +48,4 @@
         'Framework :: Django',
     ],
 )
+
```

Comment:
Not sure what the policy is about empty newlines at the end of the file, personally I prefer not doing that.